src/pages/iou/IOUCurrencySelection.js

```diff
@@ -0,0 +1,239 @@
+            <OptionRow
+                key={key}
+                mode="compact"
+                option={item}
+                onSelectRow={() => this.toggleOption(item.currencyCode)}
+                isSelected={item.currencyCode === this.state.selectedCurrency.currencyCode}
+                showSelectedState
+                hideAdditionalOptionStates
+            />
+        );
+    }
+
+    render() {
+        const sections = this.getSections();
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title="Select a currency"
+                    onCloseButtonPress={() => Navigation.goBack()}
+                />
+                <View style={[styles.flex1, styles.w100]}>
+                    <View style={[styles.flex1]}>
+                        <View style={[styles.ph5, styles.pv3]}>
+                            <TextInputWithFocusStyles
+                                styleFocusIn={[styles.textInputReversedFocus]}
+                                ref={el => this.textInput = el}
+                                style={[styles.textInput]}
+                                value={this.state.searchValue}
+                                onChangeText={(searchValue = '') => {
+                                    const {currencyOptions} = getCurrencyListForSections(
+                                        this.getCurrencyOptions(this.props.currencyList),
+                                        searchValue,
+                                    );
+                                    this.setState({
+                                        searchValue,
+                                        currencyData: currencyOptions,
+                                    });
+                                }}
+                                placeholder="Search"
+                                placeholderTextColor={themeColors.placeholderText}
+                            />
+                        </View>
+                        <View style={[styles.flex1]}>
+                            <SectionList
+                                bounces={false}
+                                indicatorStyle="white"
+                                keyboardShouldPersistTaps="always"
+                                showsVerticalScrollIndicator={false}
+                                sections={sections}
+                                keyExtractor={this.extractKey}
+                                stickySectionHeadersEnabled={false}
+                                renderItem={this.renderItem}
+                                renderSectionHeader={({section: {title}}) => (
+                                    <View>
+                                        <Text style={[styles.p5, styles.textMicroBold, styles.colorHeading]}>
+                                            {title}
+                                        </Text>
+                                    </View>
+                                )}
```

Comment:
This would be cleaner in a class-scoped function (especially since you already have `renderItem` as one).